src/codegen/templates/property_deleter_template.py.mako

```diff
@@ -0,0 +1,37 @@
+<%def name="script_property_deleter(attribute)">\
+<%
+        from codegen.utilities.text_wrappers import wrap
+    %>\
+    @${attribute.name}.deleter
+    def ${attribute.name}(self):
+<%
+        argtypes = []
+        for handle_parameter in attribute.handle_parameters:
+            if handle_parameter.ctypes_data_type == 'ctypes.c_char_p':
+                argtypes.append('ctypes_byte_str')
+            else:
+                argtypes.append(handle_parameter.ctypes_data_type)
+    %>\
+    %if len(attribute.c_function_name) < 33:
+        cfunc = lib_importer.${'windll' if attribute.calling_convention=='StdCall' else 'cdll'}.DAQmxReset${attribute.c_function_name}
+    %else:
+        cfunc = (lib_importer.${'windll' if attribute.calling_convention=='StdCall' else 'cdll'}.
```

Comment:
I would make an accessor in the attribute class for this snippet called `get_lib_importer_type` or something:

```python
${'windll' if attribute.calling_convention=='StdCall' else 'cdll'}
```

Reply:
With that, you might not even need `calling_convention` at all anymore!

Reply:
Added a helper function in `attribute` to return the import type. The mako template will make use of that helper function to determine if it is windll/ cdll call.